lib/mixins/element-mixin.html

```diff
@@ -255,7 +255,9 @@
       // ensure all gathered styles are actually in this template.
       for (let i=0; i < styles.length; i++) {
         let s = styles[i];
-        let templateStyle = templateStyles[templateStyleIndex];
+        // if no styles in the template, fall back to first element child of template
+        let templateStyle = templateStyles.length ?
+          templateStyles[templateStyleIndex] : template.content.firstElementChild;
```

Comment:
Lacking the knowledge, but I assume the `firstElementChild` represents the `style` tag represents the `.css` file?

Reply:
This just says put any non-local styles either before the closest local template style or at the top of the template

Reply:
No, the `<link>` element won't make it into the template.

Reply:
This will still throw if there's no `firstElementChild`.

Reply:
Yup, good point. Let me make the smallest possible repro.